File: src/sugtao4423/lod/ChromeIntent.java
package sugtao4423.lod;

import android.content.Context;
import android.graphics.Color;
import android.net.Uri;
import android.support.customtabs.CustomTabsIntent;

public class ChromeIntent{

	public ChromeIntent(Context context, Uri uri){
		CustomTabsIntent.Builder builder = new CustomTabsIntent.Builder();
		builder.setShowTitle(true);
		builder.enableUrlBarHiding();
		builder.addDefaultShareMenuItem();
		builder.setToolbarColor(Color.parseColor(context.getString(R.color.statusBar)));
		builder.build().launchUrl(context, uri);
	}

}

File: app/src/main/java/sugtao4423/lod/ChromeIntent.kt
package sugtao4423.lod

import android.content.Context
import android.net.Uri
import android.support.customtabs.CustomTabsIntent
import android.support.v4.content.ContextCompat

class ChromeIntent(context: Context, uri: Uri) {

    init {
        CustomTabsIntent.Builder().apply {
            setShowTitle(true)
            enableUrlBarHiding()
            addDefaultShareMenuItem()
            setToolbarColor(ContextCompat.getColor(context, R.color.statusBar))
            build().launchUrl(context, uri)
        }
    }

}